sdk/healthdataaiservices/azure-health-deidentification/CHANGELOG.md

```diff
@@ -1,14 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (2024-12-13)
```

Comment:
Update the release date after apiview approval.

Reply:
Updated